projects/bp-gallery/src/components/views/picture/sidebar/picture-info/DateRangeSelectionField.tsx

```diff
@@ -73,7 +84,11 @@ const DateRangeSelectionField = ({
         }}
         tabIndex={0}
       >
-        {timeRange ? formatTimeStamp(timeRange) : `${t('pictureFields.noTime')}`}
+        {timeRange
+          ? timeRange.isEstimate
+            ? `${t('pictureFields.around')}` + formatTimeStamp(timeRange)
+            : formatTimeStamp(timeRange)
+          : `${t('pictureFields.noTime')}`}
```

Comment:
How should I format the double ternary operator?

Reply:
Auslagern in eine Hilfsfunktion?